webapp/src/components/cardDetail/comment.tsx

```diff
@@ -44,7 +44,6 @@ const Comment: FC<Props> = (props: Props) => {
     const formattedText = 
     <Provider store={(window as any).store}>
         {messageHtmlToComponent(formatText(comment.title, {
-            renderer: Utils.getMarkdownRenderer(),
```

Comment:
Have you double checked how Markdown tables render in cards? I see that the renderer returned by `Utils.getMarkdownRenderer()` also changes how those render as well

Reply:
Yup, recheck the link and the table part which was part of `getMarkdownRenderer` and seems to be working fine.